OpenRA.Mods.D2k/Traits/Sandworm.cs

```diff
@@ -0,0 +1,139 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using System;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	class SandwormInfo : WandersInfo, Requires<MobileInfo>, Requires<RenderUnitInfo>, Requires<AttackBaseInfo>
+	{
+		[Desc("Time between rescanning for targets (in ticks).")]
+		public readonly int TargetRescanInterval = 32;
+
+		[Desc("The radius in which the worm \"searches\" for targets.")]
+		public readonly WRange MaxSearchRadius = WRange.FromCells(32);
```

Comment:
This seems way too big to me.  For reference, if you have a 1920 x 1080 px monitor and center your view on the worm, then it will be scanning out past the edges of your visible screen.